src/app/index.ts

```diff
@@ -1,2 +1,29 @@
-export * as Wallets from "./wallets"
-export * as Users from "./users"
+import * as AccountsMod from "./accounts"
+import * as AdminMod from "./admin"
+import * as LightningMod from "./lightning"
+import * as PricesMod from "./prices"
+import * as UsersMod from "./users"
+import * as WalletsMod from "./wallets"
+import { wrapAsyncToRunInSpan, wrapToRunInSpan } from "@services/tracing"
+
+const allFunctions = {
+  Accounts: { ...AccountsMod },
+  Admin: { ...AdminMod },
+  Lightning: { ...LightningMod },
+  Prices: { ...PricesMod },
+  Users: { ...UsersMod },
+  Wallets: { ...WalletsMod },
+}
+const syncFunctions = ["PaymentStatusChecker"]
+
+for (const subModule in allFunctions) {
+  for (const fn in allFunctions[subModule]) {
+    const wrapper = syncFunctions.includes(fn) ? wrapToRunInSpan : wrapAsyncToRunInSpan
+    allFunctions[subModule][fn] = wrapper({
+      namespace: "app",
```

Comment:
should we encode the namespace? `app.accounts`, `app.admin`... etc? Would be easy to add here. As keys we should be using the names from [SemanticConvetions](https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-semantic-conventions/src/trace/SemanticAttributes.ts#L340-L345)

Reply:
Updated https://github.com/GaloyMoney/galoy/pull/837/commits/fea23b3ea8b3294e33d4d6cbb202b35aac28de88